src/components/PeopleFilters.tsx

```diff
@@ -27,45 +65,26 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
+            {['16', '17', '18', '19', '20'].map(century => (
+              <Link
+                data-cy="century"
+                className={classNames('button mr-1',
+                  { 'is-info': centuries.includes(century) })}
+                to={
+                  {
+                    search: getSearchWith(searchParams, {
+                      centuries: centuries.includes(century)
```

Comment:
This nesting of code is a bit difficult to read and maintain. Therefore, it is worth taking out such chains in the definition of a separate variable and substituting the finished result, for example:
```
... .map((century) => {
  const currentCenturies = centuries.includes(century) ? centuries.filter(cent => cent !== century) : [...centuries, century];
  ...
  {
    search: getSearchWith(searchParams, { centuries: currentCenturies });
  }
  ...
});
```